net/redis/files/redis.init

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh /etc/rc.common
+
+START=98
+STOP=81
+
+SERVICE_USE_PID=1
```

Comment:
ok then, so I'll stick to the old init since I am still nor quite familiar in writing procd init scritps...